ansible/roles/vagrant/files/motd

```diff
@@ -1,31 +1,11 @@
-
 Welcome to the Pulp dev environment!
 
 Here are some tips:
 * The 3.0-dev branch is experimental, and my be broken.
 
 * Your code is all checked out inside of ~/devel/.
 
-* The default username:password is admin:admin. When your session expires, you
-  can log in again with pulp-admin login -u admin
-
-* You can type workon <project> to quickly cd to a project dir and activate its
-  virtualenv. For example:
-
-    $ workon pulp_python
-
-  Similarly, run "workon pulp" to jump into the pulp platform virtualenv.
-
-* There are a set of bash functions in your .bashrc that are useful.
-
-  Run 'phelp' to list the available functions and what they do.
-
-* phttp is an alias that gives you authenticated access to the rest API using httpie:
-
-    phttp https://localhost/pulp/api/v2/repositories/zoo/
-
-* You can ssh into your vagrant environment with vagrant ssh, but presumably
-  you already know this since you are reading this message :)
+* The 'phelp' alias will list useful functions and what they do.
```

Comment:
This message displays even if phelp isn't aliased. 
When I vagrant up the minimal-playbook `supplement_bashrc` is false and the alias.bashrc is not copied over.